### PR TITLE
Bug 1118143 - [Stingray] Display newly installed apps in AppDeck

### PR DESCRIPTION
**Pull request by dwi2**
Notice that `removing uninstall app from AppDeck` will be fixed in bug 1115633 instead of here.

https://bugzilla.mozilla.org/show_bug.cgi?id=1118143

### CODE REVIEW

**Comment by try-server-hook** [comment]
[dwi2 Tzu-Lin Huang (dwi2)](https://github.com/dwi2) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=02e35565f600)

**Comment by try-server-hook** [comment]
[dwi2 Tzu-Lin Huang (dwi2)](https://github.com/dwi2) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=33a65715f745)

**Comment by try-server-hook** [comment]
[dwi2 Tzu-Lin Huang (dwi2)](https://github.com/dwi2) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4f6bb318b251)

File: tv_apps/app-deck/js/app_deck.js [hunk 1]
```diff
@@ -61,9 +61,27 @@
...
         that._contextMenu = new ContextMenu();
         that._contextMenu.init(that);
+        Applications.on('install', that.onAppInstalled.bind(that));
+        Applications.on('update', that.onAppUpdated.bind(that));
+        Applications.on('uninstall', that.onAppUninstalled.bind(that));
       });
     },
 
+    _fillAppButtonIcon: function ad_fillAppButtonIcon(app, elem) {
+      Applications.getIconBlob(app.manifestURL, app.entryPoint, 180,
```
Comment (huchengtw-moz) [inline tv_apps/app-deck/js/app_deck.js]: Why do we hardcode a size here? We should create a const with window.devicePixelRatio applied. I found that our home app also had similar issue. Please update the code here and file a bug for smart-home app.

**Comment by try-server-hook** [comment]
[dwi2 Tzu-Lin Huang (dwi2)](https://github.com/dwi2) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dfa208badc85)

**Comment by try-server-hook** [comment]
[dwi2 Tzu-Lin Huang (dwi2)](https://github.com/dwi2) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=9930dcd4dcf6)

**Pull request merged**